pkg/ddl/ttl.go

```diff
@@ -30,9 +30,6 @@ import (
 	"github.com/pingcap/tidb/pkg/util/dbterror"
 )
 
-// DefaultTTLJobInterval is the default value for ttl job interval.
-const DefaultTTLJobInterval = "1h"
```

Comment:
This const is moved to `meta` package:  https://github.com/pingcap/tidb/pull/57758/files#diff-1730439e8c1b7d1a7019391006a2c0cb2f455b9de48769c9b6097c9c0a2b995eR1324